### ISSUE TITLE
Module not found: Package path ./unstable-native-dependencies is not exported from package

### ISSUE THREAD

**Issue by f00d4tehg0dz** [3DJakob/react-tinder-card#122]
After upgrading to react-tinder-card both v1.6.0 and v1.6.1 with @react-spring/web@9.5.5 I'm experiencing this error after running `npm run build ` 

```
./node_modules/react-panresponder-web/dist/panresponder.js
Module not found: Package path ./unstable-native-dependencies is not exported from package C:\Users\xxx\xxx\Documents\GitHub\xxx\node_modules\react-dom (see exports field in C:\Users\xxx\xxx\Documents\GitHub\xxx\node_modules\react-dom\package.json)
```

**package.json**
` "dependencies": {
    "next": "^12.3.1",
    "react": "^18.0.0",
    "react-dom": "18.0.0",
    "react-tinder-card": "^1.6.1"
  },`

**Comment by CodeOne45** [comment]
I'm facing the same issue :(

**Comment by f00d4tehg0dz** [comment]
Hey @CodeOne45  I created a new project from scratch and haven't had an issue.  I suspect something else is going on with my existing project. I'll leave this ticket open to provide any insights once I finish analyzing

**Comment by f00d4tehg0dz** [comment]
Looks like we all are using the same [github](https://github.com/CleverProgrammers/tinder-blockchain-youtube/issues/3)  and facing the same issue. I suspect this isn't an issue on your end @3DJakob

**Comment by strangecarr** [comment]
yep same with me, I updated to 1.6.1 on an existing project and getting the same error

**Comment by Develoopment** [comment]
Same here, I created the same project with a previous version and it works but this version outputs said error.

**Comment by 3DJakob** [comment]
It seems like a lot of people have this issue. However, I can not seem to reproduce this. Are people running native or web?

Please try to clone the [web demo](https://github.com/3DJakob/react-tinder-card-demo) or [native demo](https://github.com/3DJakob/react-native-tinder-card-demo) and reproduce the issue there. Both repositories are running 1.6.1 so the issue should be present there.

**Comment by 3DJakob** [comment]
If running expo perhaps you need a newer expo version 🤔 Other fixes could include trying to update your node version before installing.

**Comment by strangecarr** [comment]
I'm running web version, just updated to node 18.12 but unfortunately still getting same error. Will have a go at cloning the web demo you linked to.

**Comment by f00d4tehg0dz** [comment]
Hey, @3DJakob  I've made a git repo with the latest nextjs template `npx create-next-app@latest` with your web src/example advanced/simple. 

https://github.com/f00d4tehg0dz/my-app

Commands ran are 
```
npm i
npm run build
```

(fails during build) with 
```
info  - Creating an optimized production build
Failed to compile.

./node_modules/react-panresponder-web/dist/panresponder.js
Module not found: Package path ./unstable-native-dependencies is not exported from package C:\Users\adria\Downloads\my-app\node_modules\react-dom (see exports field in C:\Users\adria\Downloads\my-app\node_modules\react-dom\package.json)

Import trace for requested module:
./node_modules/react-panresponder-web/dist/panresponder.js
./node_modules/react-tinder-card/index.js
./components/Advanced.js
```

**Comment by CodeOne45** [comment]
> Hey @CodeOne45 I created a new project from scratch and haven't had an issue. I suspect something else is going on with my existing project. I'll leave this ticket open to provide any insights once I finish analyzing
> 
> _ninja edit_ after placing the card component in the new project, it fails. Digging further into this the react-panresponder-web package is causing the issue. It was raised here: [arqex/react-panresponder-web#1](https://github.com/arqex/react-panresponder-web/issues/1) So unless @3DJakob uses a different package, we may just need to revert to an older version.

@f00d4tehg0dz thank's for your replay, however, even after downgrading to the previous version I still face the error on a server. But, still works locally.

**Comment by aseprahman04** [comment]
@3DJakob  try using react-script more than version 3, on the example using react-script v3 and also upgrade using react v18 , hopefully you can see the error.

**Comment by aseprahman04** [comment]
@CodeOne45 @f00d4tehg0dz  try using v 1.5.0 and react spring 9.5.5 on my local the error gone

**Comment by CodeOne45** [comment]
@aseprahman04 did what you said but still facing the same error 😫. (cf: https://github.com/CodeOne45/discover-mesic/actions/runs/3337422545/jobs/5523744355)

**Comment by CodeOne45** [comment]
@aseprahman04 yes I tried but still facing the same error even in my local. :(

**Comment by BotJB** [comment]
> Also ran into the same issue. It didn't work on 1.6.1 but it works find when I downgrade to 1.5.4
> 
> I'm on: next 12.3.1, react 18.2.0, @react-spring/web 9.5.5

I am still facing the same issue despite running these versions help please

**Comment by mattg66** [comment]
Same issue here

**Comment by 3DJakob** [comment]
I'm well aware of this issue and working on a fix! For now, if you experience this issue please use version 1.5.4.

`npm install react-tinder-card@1.5.4`

I will post in this thread once I release the new version which fixes this issue.

**Comment by 3DJakob** [comment]
@f00d4tehg0dz @CodeOne45 @strangecarr @Develoopment @aseprahman04 @sipec @BotJB @mattg66 This has been fixed in PR https://github.com/3DJakob/react-tinder-card/pull/124 and released in v1.6.2 🎉

Please run `npm install react-tinder-card@1.6.2` to resolve the issue.
